UnitySDK/Assets/ML-Agents/Scripts/Policy/BehaviorParameters.cs

```diff
@@ -70,7 +71,7 @@ public IPolicy GeneratePolicy(Func<float[]> heuristic)
                 case BehaviorType.Default:
                     if (FindObjectOfType<Academy>().IsCommunicatorOn)
                     {
-                        return new RemotePolicy(m_BrainParameters, behaviorName);
+                        return new RemotePolicy(m_BrainParameters, m_BehaviorName);
```

Comment:
```suggestion
                        return new RemotePolicy(m_BrainParameters, behaviorName);
```
maybe a bad merge, needs to be behaviorName (the property)